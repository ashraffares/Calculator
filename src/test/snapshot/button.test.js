import renderer from 'react-test-renderer';
import Button from '../../component/button';

describe('Button component', () => {
  it('matches the snapshot', () => {
    const handleClick = () => 'just for testing';
    const button = renderer.create(<Button clickHandler={handleClick} name="1" />).toJSON();
    expect(button).toMatchSnapshot();
  });
});
