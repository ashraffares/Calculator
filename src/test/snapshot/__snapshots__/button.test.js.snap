// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Butt component matches the snapshot 1`] = `
<button
  className="btn orange NormalSize"
  onClick={[Function]}
  type="button"
>
  1
</button>
`;

exports[`ButtonPanel component matches the snapshot 1`] = `
<button
  className="btn orange NormalSize"
  onClick={[Function]}
  type="button"
/>
`;
